requirements/base.txt

```diff
@@ -42,7 +42,7 @@ packaging==21.3
     # via drf-yasg
 pyparsing==3.0.9
     # via packaging
-pytz==2022.2.1
+pytz==2022.4
```

Comment:
No breaking change affecting the current implementation https://github.com/stub42/pytz/commit/8eeefc360913ce56f8759c33b31e00542a0d3078